eo-maven-plugin/src/main/java/org/eolang/maven/util/Home.java

```diff
@@ -124,10 +85,11 @@ public void save(final String str, final Path path) throws IOException {
      *
      * @param txt Text
      * @param path Cwd-relative path to file
+     * @return Path to saved file
      * @throws IOException If fails
      */
-    public void save(final Text txt, final Path path) throws IOException {
-        this.save(new InputOf(txt), path);
+    public Path save(final Text txt, final Path path) throws IOException {
```

Comment:
@volodya-lombrozo Try to avoid commands which returns anything. Please check https://www.yegor256.com/2018/08/22/builders-and-manipulators.html:

> We either receive objects or tell them what to do. We must not mix those activities in one method

Reply:
@mximp done